integrationTests/vm/txsFee/builtInFunctions_test.go

```diff
@@ -0,0 +1,195 @@
+package txsFee
+
+import (
+	"encoding/hex"
+	"math/big"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/vmcommon"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBuildInFunctionChangeOwnerCallShouldWork(t *testing.T) {
+	t.Parallel()
+
+	testContext := vm.CreatePreparedTxProcessorWithVMs(true)
+	defer testContext.Close()
+
+	scAddress, owner := doDeploy(t, &testContext)
+	testContext.TxFeeHandler.CreateBlockStarted()
+
+	newOwner := []byte("12345678901234567890123456789112")
+	gasPrice := uint64(10)
+	gasLimit := uint64(1000)
+
+	txData := []byte(core.BuiltInFunctionChangeOwnerAddress + "@" + hex.EncodeToString(newOwner))
+	tx := vm.CreateTransaction(1, big.NewInt(0), owner, scAddress, gasPrice, gasLimit, txData)
+	_, err := testContext.TxProcessor.ProcessTransaction(tx)
+	require.Nil(t, err)
+	require.Nil(t, testContext.GetLatestError())
+
+	_, err = testContext.Accounts.Commit()
+	require.Nil(t, err)
+
+	checkOwnerAddr(t, &testContext, scAddress, newOwner)
+
+	expectedBalance := big.NewInt(88180)
+	vm.TestAccount(t, testContext.Accounts, owner, 2, expectedBalance)
+
+	// check accumulated fees
+	accumulatedFee := testContext.TxFeeHandler.GetAccumulatedFees()
```

Comment:
You can search everywhere after this message: // check accumulated fees , and replace the below variable name from 
accumulatedFee to accumulatedFees

Reply:
done.